testpmd-lb-operator/roles/loadbalancer/templates/deployment.yml

```diff
@@ -26,6 +26,10 @@ spec:
 {% endfor %}
 {% endfor %}
         ]'
+{% if runtimeClassName is defined and runtimeClassName|length %}
```

Comment:
I think we're not providing runtimeClassName in LoadBalancer CR creation, we just do it for TestPMD CR. In that case, we need to add it in https://github.com/rh-nfv-int/nfv-example-cnf-deploy/blob/master/roles/example-cnf-app/templates/lb-cr.yaml.j2

Reply:
My bad, I've seen you did the change, closing this